libfwupd/fwupd.map

```diff
@@ -707,5 +707,6 @@ LIBFWUPD_1.7.0 {
     fwupd_security_attr_add_guids;
     fwupd_security_attr_get_guids;
     fwupd_security_attr_has_guid;
+    fwupd_security_attr_flag_to_string_array;
```

Comment:
This is the wrong place for it (see CI failure)

Reply:
Yes, I am trying to resolve this. I may let this function live in src/.